src/zck_read_header.c

```diff
@@ -180,6 +179,13 @@ int main (int argc, char *argv[]) {
             if (!digest_uncompressed)
                 digest_uncompressed = "";
 
+	    if (chk == zck_get_first_chunk(zck)) {
+		    printf("STRLEN > = %d\n", strlen(digest_uncompressed) > 0);
+        	    printf("       Chunk Checksum %*c        Start    Comp size         Size\n",
```

Comment:
It would be nice to have a field header "Uncompressed Chunk Checksum" when the flag is set.

Reply:
Done.